industrial_ci/CMakeLists.txt

```diff
@@ -2,5 +2,5 @@ cmake_minimum_required(VERSION 2.8.3)
 project(industrial_ci)
 find_package(catkin REQUIRED)
 catkin_package()
-install(FILES check_metapackage.py ci_main.sh
```

Comment:
metapackage removal was introduced for this thoughts https://github.com/jsk-ros-pkg/jsk_travis/issues/117, which honestly I'm not yet sure what is the problem. So, we *may* see the same issue with that ticket but let's face it.